app.lua

```diff
@@ -46,4 +46,9 @@ app:match("changepackage", "/changepackage", respond_to(changepackage))
 app:match("verifyemail", "/verifyemail", respond_to(verify_email))
 app:match("resendverification", "/resendverification", respond_to(resend_verification))
 
+if config._name == "development" or config.name == "docker" or config.name == "kubernetes" then
```

Comment:
did you mean to use config._name for the first one and config.name for the others?

Reply:
That's a typo. Fixed locally, but running db calls from this context is no-go.